tools/common/Target.cs

```diff
@@ -205,9 +205,9 @@ public void GatherFrameworks ()
 							}
 							break;
 						case "WatchKit":
-							// Xcode 11 beta 1 doesn't ship WatchKit for iOS
+							// Xcode 11 doesn't ship WatchKit for iOS
 							if (Driver.XcodeVersion.Major == 11 && App.Platform == ApplePlatform.iOS) {
-								ErrorHelper.Warning (99, "Not linking with WatchKit because Xcode 11 beta 1 does not ship with the WatchKit framework.");
+								ErrorHelper.Warning (5219, "Not linking with WatchKit because it has been removed from iOS.");
```

Comment:
does the condition still makes sense ? as we'll never link with WatchKit anymore (whatever Xcode version is being used)

Reply:
It shouldn't ever be hit, but _shouldn't_ and _won't_ are quite far apart in the dictionary.